services/characterlcdscreen.md

```diff
@@ -0,0 +1,49 @@
+# Character LCD Screen
+
+    identifier: 0x1f37c56a
+
+A character LCD screen.
+
+## Registers
+
+    rw message: string @ value
+
+Sets text, with `\n` as the newline character.
+
+    rw cursor: bool @ 0x80
+
+Configures to show or hide the cursor.
+
+    rw blink: bool @ 0x81
+
+Configures to blink the cursor.
+
+    enum TextDirection : u8 {
+        LeftToRight = 1,
+        RightToLeft = 2
+    }
+    rw text_direction: TextDirection @ 0x82
+
+Specifies the RTL or LTR direction of the text.
+
+    rw backlight_color? @ 0x83 {
+        red: u8
+        green: u8
+        blue: u8
+    }
+
+If available, sets the backlight color.
```

Comment:
should we do HSV, so that it's clear V is used when backlight is monochromatic?

Reply:
I guess this is backlight color or LED color, in case it's an LED display

Reply:
Something like this would allow me to mount better UIs with color pickers
```
rw backlight_color: u24 hsv @ 0x180
```

Reply:
I think we could use `hsv` as type; we don't really have `u24` type.